tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/common/UserTaskRecordToTaskEntityMapper.java

```diff
@@ -62,6 +62,9 @@ public Optional<TaskEntity> map(final Record<UserTaskRecordValue> record) {
 
     final UserTaskRecordValue recordValue = record.getValue();
     final String processDefinitionId = String.valueOf(recordValue.getProcessDefinitionKey());
+    System.out.println("processDefinitionVersion: " + recordValue.getProcessDefinitionVersion());
```

Comment:
Forgotten SOUTs :)

Reply:
I am applying one extra change :) Gonna remove it in the next commit